teps/0091-trusted-resources.md

```diff
@@ -265,61 +245,7 @@ code.
 
-Option 1: Configure the keys in configmap. The configmap cannot support a list of keys but one solution is that we can allow multiple keys be stored in one string and when fetching the keys we can parse it to get the list of keys. The string may follow [JSONPath](https://tools.ietf.org/id/draft-goessner-dispatch-jsonpath-00.html)  syntax of array. The con is that simple typo may cause parsing errors.
-
-Option 2: Add a new field `keyref` to refer to the key path
-```yaml
-kind: TaskRun
-apiVersion: tekton.dev/v1beta1
-metadata:
- name: example-tr
-spec:
- taskRef:
-   name: example-task
-   keyref: keypath # this defines the public key of this taskref
-```
-
-```yaml
-kind: PipelineRun
-apiVersion: tekton.dev/v1beta1
-metadata:
- name: example-pr
-spec:
- pipelineRef:
-   name: example-pipeline
-   keyref: keypath # this defines the public key of this pipelineref
-```
-
-```yaml
-apiVersion: tekton.dev/v1beta1
-kind: Pipeline
-metadata:
- annotations:
-   tekton.dev/signature: MEQCIHhFC480mv7mh/6dxAlp/mvuvGXanuSghMsT+iBhWxt5AiBDvHv8sfKjJ3Ozrzvp+cjr30AOx1SPQDCcaJjlpAoVrA==
- name: example-Pipeline
-spec:
- tasks:
-   - name: example-task
-     taskRef:
-       name: example-task1
-       keyref: keypath1  # this defines the public key of this taskref
-   - name: example-task2
-     taskRef:
-       name: example-task2
-       keyref: keypath2  # this defines the public key of this taskref
-```
-`keyref` defines the path of public key of this taskRef. All the tasks within a pipeline need to be signed off, and they may come from different sources with different public keys.
-So a `keyref` is needed when signing the pipeline. The signing cli doesn't need to verify the referred keys when signing pipeline.
-
-**Pros vs Cons:**
-| Option | Pros | Cons |
-| -------- | ----------- | ----------- |
-| Config at install level (e.g. configmap) | Easy to config at a central place |  ConfigMap doesn't support list, may need to parse the string or create a new cluster level config to store the keys
-| Add keyref in taskRef&pipelineRef | Each resource can speficy the corresponding key, no need to loop all available keys for one resource's verification| Introduce cost for users to write and maintain YAML files
-
-In this TEP we proposed to choose option 1, for option 2 it would cause a bad user experience that if one pipeline contains a bunch of tasks, then users will need to configure them manually one by one. This would be a big cost for using this feature.
-
-For option 1, we mainly want to specify a list of trusted keys in Tekton Pipeline, and when verifying we can loop the keys to see if the resource can pass any of these keys.
+In this TEP we proposed to choose store the public keys at installation-level. For example, the keys can be configured in a new CRD and deployed at cluster. This can help to support multiple keys, validate the spec, and have dedicated RBAC policies.
```

Comment:
grammar nit: "...we propose to store the public keys..."

Reply:
Thank you! I will change all other places as well